packages/analytics/analytics-utilities/src/types/explore/requests.ts

```diff
@@ -45,18 +57,37 @@ export const filterableRequestMetrics = makeFilterable(queryableRequestMetrics)
 
 export type FilterableRequestMetrics = typeof filterableRequestMetrics[number]
 
-export interface RequestDimensionFilter extends Omit<ExploreFilter, 'dimension'> {
-  dimension?: FilterableRequestDimensions
-  field?: FilterableRequestDimensions
+export interface RequestInFilter {
+  type: ExploreFilterTypesV2
+  field: FilterableRequestDimensions | FilterableRequestWildcardDimensions | FilterableRequestMetrics
+  value: (string | number)[]
+}
+export interface RequestEqualsFilter {
+  type: RequestFilterTypeEqualsV2
+  field: FilterableRequestDimensions | FilterableRequestWildcardDimensions
+  value: string
 }
-
 export interface RequestMetricFilter {
-  type: MetricFilterTypesV2
+  type: RequestFilterTypeMetricV2
   field: FilterableRequestMetrics
-  value: number | null
+  value: number
+}
+export interface RequestEmptyFilter {
+  type: RequestFilterTypeEqualsV2
+  field: FilterableRequestDimensions | FilterableRequestWildcardDimensions | FilterableRequestMetrics
+}
+export interface RequestWildcardFilter {
+  type: RequestFilterTypeWildcardV2
+  field: FilterableRequestWildcardDimensions
+  value: string
 }
 
-export type RequestFilter = RequestDimensionFilter | RequestMetricFilter
+export type RequestFilter = ExploreFilter |
```

Comment:
we still support old field object with `dimension` and `values`, but not for any of the new filterable dimension.